.github/scripts/build_publish_nightly_docker.sh

```diff
@@ -19,14 +19,25 @@ make -f docker.Makefile \
 PYTORCH_NIGHTLY_COMMIT=$(docker run \
        ghcr.io/pytorch/pytorch-nightly:${PYTORCH_DOCKER_TAG} \
        python -c 'import torch; print(torch.version.git_version)' | head -c 7)
+
 docker tag ghcr.io/pytorch/pytorch-nightly:${PYTORCH_DOCKER_TAG} \
-       ghcr.io/pytorch/pytorch-nightly:${PYTORCH_NIGHTLY_COMMIT}
+       ghcr.io/pytorch/pytorch-nightly:${PYTORCH_NIGHTLY_COMMIT}-cu${CUDA_VERSION}
```

Comment:
Should we add cuda version in image name or image tag?

Reply:
No I think having one image repository point to multiple tags should be fine.